Makefile

```diff
@@ -50,6 +50,17 @@ endif
 	go generate ./pkg/... ./cmd/...
 	go run ./hack/gen-openapi-spec/main.go > ${CURRENT_DIR}/api/openapi-spec/swagger.json
 
+# kubebuilder binary
+kubebuilder:
+ifeq (, $(shell which kubebuilder))
+	# Download kubebuilder and extract it to tmp
+	curl -sL https://go.kubebuilder.io/dl/2.0.0/$(shell go env GOOS)/$(shell go env GOARCH) | tar -xz -C /tmp/
```

Comment:
Kruise is generated by kubebuilder 1.x, and has not supported kubebuilder 2.x yet. Maybe we should install 1.0.8 instead of 2.0.0 .

Reply:
Ah, good catch. I meant to install 1.0.8 but makes a mistake in copy/paste. Will change soon.

Reply:
done